### ISSUE TITLE
Failed to execute "renku update on a test dataset"

### ISSUE THREAD

**Issue by grzeskokbol** [SwissDataScienceCenter/renku-python#1529]
Hi,

I fail to execute the "renku update" command on the test dataset and files related to the tutorial:
 https://renku.readthedocs.io/en/latest/tutorials/first_steps/06_create_workflow.html

using my local machine:

## Details

**Renku version:** 0.11.1
**OS:** Linux (#51-Ubuntu SMP Fri Sep 4 19:50:52 UTC 2020)
**Python:** 3.8.2

### Traceback

```
Traceback (most recent call last):
  File "[...]/renku/cli/exception_handler.py", line 117, in main
    result = super().main(*args, **kwargs)
  File "[...]/renku/cli/exception_handler.py", line 87, in main
    return super().main(*args, **kwargs)
  File "[...]/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "[...]/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "[...]/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "[...]/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "[...]/renku/core/commands/client.py", line 102, in new_func
    result = ctx.invoke(method, client, *args, **kwargs)
  File "[...]/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "[...]/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "[...]/renku/cli/status.py", line 54, in status
    status = graph.build_status(revision=revision, can_be_cwl=no_output)
  File "[...]/renku/core/commands/graph.py", line 333, in build_status
    current_files = self.build(dependencies=dependencies, can_be_cwl=can_be_cwl,)
  File "[...]/renku/core/commands/graph.py", line 306, in build
    dependencies = self.process_dependencies(dependencies, visited=ignore)
  File "[...]/renku/core/commands/graph.py", line 247, in process_dependencies
    activity = processing.client.process_commit(processing.commit)
  File "[...]/renku/core/management/repository.py", line 262, in process_commit
    process = Activity.from_jsonld(yaml.safe_load(data), client=self, commit=commit)
  File "[...]/renku/core/models/provenance/activities.py", line 373, in from_jsonld
    return schema(client=client, commit=commit, flattened=True).load(data)
  File "[...]/marshmallow/schema.py", line 723, in load
    return self._do_load(
  File "[...]/marshmallow/schema.py", line 905, in _do_load
    raise exc
marshmallow.exceptions.ValidationError: {'http://schema.org/isPartOf': {'http://schema.org/creator': {'http://www.w3.org/2000/01/rdf-schema#label': ['Not a valid string.'], 'http://schema.org/name': ['Not a valid string.']}}, 'http://www.w3.org/ns/prov#wasAssociatedWith': {'http://www.w3.org/2000/01/rdf-schema#label': ['Not a valid string.'], 'http://schema.org/name': ['Not a valid string.']}, 'http://www.w3.org/ns/prov#qualifiedUsage': {'http://www.w3.org/ns/prov#entity': {'http://schema.org/isPartOf': {'http://schema.org/creator': {'http://www.w3.org/2000/01/rdf-schema#label': ['Not a valid string.'], 'http://schema.org/name': ['Not a valid string.']}}}}, 'http://www.w3.org/ns/prov#activity': {'http://www.w3.org/ns/prov#qualifiedGeneration': {'http://schema.org/isPartOf': {'http://schema.org/creator': {'http://www.w3.org/2000/01/rdf-schema#label': ['Not a valid string.'], 'http://schema.org/name': ['Not a valid string.']}}}}, 'http://www.w3.org/ns/prov#qualifiedAssociation': {'http://www.w3.org/ns/prov#hadPlan': {'http://schema.org/isPartOf': {'http://schema.org/creator': {'http://www.w3.org/2000/01/rdf-schema#label': ['Not a valid string.'], 'http://schema.org/name': ['Not a valid string.']}}, 'https://swissdatasciencecenter.github.io/renku-ontology#hasOutputs': {'https://swissdatasciencecenter.github.io/renku-ontology#produces': {'http://schema.org/isPartOf': {'http://schema.org/creator': {'http://www.w3.org/2000/01/rdf-schema#label': ['Not a valid string.'], 'http://schema.org/name': ['Not a valid string.']}}}}, 'https://swissdatasciencecenter.github.io/renku-ontology#hasInputs': {'https://swissdatasciencecenter.github.io/renku-ontology#consumes': {'http://schema.org/isPartOf': {'http://schema.org/creator': {'http://www.w3.org/2000/01/rdf-schema#label': ['Not a valid string.'], 'http://schema.org/name': ['Not a valid string.']}}}}}}}
```

**Comment by Panaetius** [comment]
Thank you for letting us know. We were aware of this issue in a different context, but thought it only affected master, not 0.11.1. 

Our guess is that you changed your name in git at some point, is that correct? We recently switched to storing our metadata in a different format, but unfortunately overlooked this issue where having multiple names associated with the same email address would cause name conflicts in the metadata.

This bug is fixed in current master, but unfortunately there is now way to fix it/work around it on your end at the moment. You could install current master and that should work fine, though. Alternatively you could try changing your git name back, but that might also necessitate resetting your repository to before the name change.

It should be fixed if we make a new release, we are currently discussing when this would be, as we also want to get another bug fix in.

**Comment by Panaetius** [comment]
We released renku 0.11.2 last week. Could you check if this issue still exists?

**Comment by grzeskokbol** [comment]
Thank you for your message and sorry for such a late reply of mine. Indeed, multiple names associated with a single email 
was the problem as I recall using different names (due to different machines that I use) to commit the code. With the newest release, this particular problem seems not to be a case anymore for me.

Thanks a lot !
